lib/battlestation/cli.rb

```diff
@@ -164,45 +172,74 @@ def install_packages(current_dirname)
       }
     end
 
-    def verify_rbenv
```

Comment:
This was adding minimal value, and was failing on the first run since the environment is not fully set up.